src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java

```diff
@@ -133,4 +129,47 @@ public static void enterTemporaryBucketName(String bucket) throws IOException {
   public static void clickPreviewData() {
     SeleniumHelper.waitAndClick(CdfBigQueryPropertiesLocators.previewData);
   }
+
+  public static void enterPartitionField(String partitionField) {
+    CdfBigQueryPropertiesLocators.partitionField.sendKeys(partitionField);
```

Comment:
can we use `ElementHelper` in all the actions?

Reply:
This applies to all GCS actions as well.

Reply:
Element helper has additional checks and steps which are not required in all actions.
Element helper should be called only when necessary.

Reply:
Why we don't want these additional checks here? From my understanding, it checks for presence of element and scrolls to the element correctly which makes the test more stable, isn't it??

Reply:
@poojantcs WDYT?

Reply:
Hi @itsankit-google , we should use the ElementHelper as much as we can. For instances where the ElementHelper is not working, we should look to either enhance the existing functions or add new ones to it.